02-generalize/Square.cpp

```diff
@@ -0,0 +1,67 @@
+/*Program illustrates the drawing of a Square Box  in C++ programming*/
```

Comment:
Try not to use obvious comments. If you really have to use it, make it very short & simple. Consider:

`/* Square pattern generating class */`